man/systemd-nspawn.xml

```diff
@@ -719,6 +719,15 @@
       </varlistentry>
 
       <varlistentry>
+        <term><option>--syscall-whitelist</option></term>
+
+        <listitem><para><command>systemd-nspawn</command> sets up a syscall filter
+        with seccomp blocking system calls to isolate the container.
```

Comment:
"seccomp" should either be explained, or at least placed in `"`, to indicate that is not a proper word, but some specific technology...